voltaire_bundler/event_bus_manager/endpoint.py

```
@@ -112,70 +139,209 @@ async def _handle_request_cb(
+        except (asyncio.IncompleteReadError, OSError) as exc:
+            self._teardown(exc)
+        except Exception as exc:
+            logging.exception("IPC client read loop crashed for %s", self.server_id)
+            self._teardown(exc)
+
+    def _teardown(self, exc: BaseException) -> None:
+        if self._writer is not None:
+            try:
+                self._writer.close()
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)